operators/pkg/instctrl/containers_test.go

```diff
@@ -107,12 +110,14 @@ var _ = Describe("Generation of the container based instances", func() {
 			Spec: clv1alpha2.InstanceSpec{
 				Running:  true,
 				Template: clv1alpha2.GenericRef{Name: templateName, Namespace: templateNamespace},
+				Tenant:   clv1alpha2.GenericRef{Name: tenantName},
 			},
 		}
 		environment = clv1alpha2.Environment{
-			Name:            environmentName,
-			EnvironmentType: clv1alpha2.ClassContainer,
-			Image:           image,
+			Name:                environmentName,
+			EnvironmentType:     clv1alpha2.ClassContainer,
+			Image:               image,
+			MountPersonalVolume: false,
```

Comment:
Could set to true (and set valid values for the server and the path), to also test your feature.

Reply:
Change to true and assign valid names to server and path